data/card_catalogue.json

```diff
@@ -525,15 +577,15 @@
           "upgrade": "",
           "tier": "blue",
           "details": {
-            "rules": "Allows you to throw any non-magic object you can fit in your hand for double your normal ranged damage.",
+            "rules": "Allows you to throw any non-magic object you can fit in your hand as your ranged attack to deal an additional 6 damage.",
```

Comment:
IMO most cards should have linear scaling, doubling or tripling means that the card is either useless or so strong that everything else is pointless